### PR TITLE
organising gui

### PR DESCRIPTION
**Pull request by dekespo**

### CODE REVIEW

**Comment by dekespo** [comment]
more will come now @naslundx

**Comment by naslundx** [comment]
Great work :+1:

**Comment by dekespo** [comment]
OK, I have done many things and I need to point out a few things.

Do you think we need to do something about the commits above here?

File: constants.py [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+    width, height = mainWindow.width * 2 // 3, mainWindow.height * 4 // 5 
+
+class buttonsWindow:
+    width, height = mainWindow.width * 4 // 5, mainWindow.height * 1 // 5
+
+class textWindow:
+    width, height = mainWindow.width * 1 // 5, mainWindow.height * 1 // 5
+
+class plotWindow:
+    width, height = mainWindow.width * 1 // 3, mainWindow.height * 4 // 5
```
Comment (dekespo) [inline constants.py:28]: I have created these classes like that. Maybe there is a better way to configure these values. Do you know any?

Also, there is a bug, which somewhat mismatch in the sizes of the width. If you increase the width size of the main window while running, you'll see some more space is coloured.

File: gui/buttons.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import tkinter as tk
+import constants as CONST
+
+class buttonGUI(tk.Frame):
+    def __init__(self, parent):
+        buttonFrame = tk.Frame(master = parent, bg = "yellow", width = CONST.buttonsWindow.width, height = CONST.buttonsWindow.height)
+        #text = tk.StringVar()
+        #label = tk.Label(buttonFrame, textvariable = text)
+        #text.set("Buttons Part")
+        #label.pack(anchor = tk.CENTER)
```
Comment (dekespo) [inline gui/buttons.py]: I left these commented lines like that. They are supposed to show a label to clarify what the frame is supposed to do. But when I run it, the background colour just disappears. I couldn't figure it out and let me know if it works in your computer.

File: main.py [hunk 3]
```diff
@@ -1,7 +1,9 @@
-import gui as GUI
+import sys
+sys.path.append("./gui")
```
Comment (dekespo) [inline main.py:2]: I've moved and separated all the gui classes into the directory called gui (obviously :D ) , but I am not sure if this way is a good way to import a python file from another directory. What do you think?

File: constants.py [hunk 4]
```diff
@@ -4,15 +4,23 @@ class mainWindow:
...
 
 class textWindow:
     width, height = mainWindow.width * 1 // 5, mainWindow.height * 1 // 5
+    rowOrder, colOrder = 1, 1
+    background = "purple"
 
 class plotWindow:
     width, height = mainWindow.width * 1 // 3, mainWindow.height * 4 // 5
+    rowOrder, colOrder = 0, 1
+    background = "blue"
```
Comment (dekespo) [inline constants.py:28]: The bug seems that the grid pack doesn't care how much sizes you give to each one :/

**Comment by dekespo** [comment]
@naslundx good news!  I have prepared our very GUI and they are ready to do show their own magic :D . There is a funny bug about the plots, but I kind of like it, I suggest not to fix it anytime soon :) 

I will also merge this now and add issues.

**Pull request merged**